src/Payload.php

```diff
@@ -32,14 +32,19 @@ class Payload extends Constants {
     /**
      * Constructs a Payload object, contains an array in which event parameters are stored
      *
-     * @param string|null $tstamp - Timestamp for event
+     * @param int|null $timestamp_in_ms - Unix timestamp in milliseconds
      */
-    public function __construct($tstamp = NULL) {
+    public function __construct($timestamp_in_ms = NULL) {
         // Construct a name-value pairs array
         $this->nv_pairs = array();
 
         // Add Time Stamp to array on event creation
-        $this->add("dtm", ($tstamp != NULL) ? $tstamp : $_SERVER['REQUEST_TIME'] * 1000);
+        if ($timestamp_in_ms == NULL) {
+            $this->add("dtm", $_SERVER['REQUEST_TIME'] * 1000);
+        }
+        else {
+            $this->add("ttm", $timestamp_in_ms);
```

Comment:
I think we should still set the `dtm` but also set the `ttm` when one is injected.